l1-contracts/contracts/governance/PermanentRestriction.sol

```diff
@@ -19,11 +19,6 @@ import {IAdmin} from "../state-transition/chain-interfaces/IAdmin.sol";
 
 import {IPermanentRestriction} from "./IPermanentRestriction.sol";
```

Comment:
There are many unused imports: InvalidSelector, InvalidAddress, NotEnoughGas and etc

Reply:
Yeah, sorry lints dont pass anyway due to issues unrelated to this PR, so it is a bit hard to preemptively fix such things

Reply:
Will fix